cdap-ui/app/features/adapters/controllers/create-ctrl.js

```diff
@@ -231,7 +233,7 @@ angular.module(PKG.name + '.feature.adapters')
           delete sink.properties[key];
         }
       });
-      for (var i=0; i<trans.length; i++) {
+      for (i=0; i<trans.length; i++) {
```

Comment:
Spacing is not consistent all over the place.
I'm going to break it down by category, and just apply a certain set of changes per PR.
So, not everything will go in this PR.